`make check` shows errors

# Issue description

`make check` shows errors. It shouldn't – we should either tweak linters configurations or fix code.

# Environment

* Gosh version (commit hash if unreleased): `v0.0.1`
* `go env` output: not relevant.

# Minimal test code / Steps to reproduce the issue

1. Run `make check`.

# What's the actual result? (include panic message & call stack if applicable)

```$ make check
go generate ./...
go install -v ./...
go run misc/check_license.go
golangci-lint run ./...
main.go:132: G304: Potential file inclusion via variable (gosec)
		f, err := os.Open(historyFilename)
main.go:170: G304: Potential file inclusion via variable (gosec)
	b, err := ioutil.ReadFile(filename)
internal/gofuzz/corpus.go:11: G505: Blacklisted import `crypto/sha1`: weak cryptographic primitive (gosec)
	"crypto/sha1"
internal/gofuzz/corpus.go:42: G401: Use of weak cryptographic primitive (gosec)
	name := fmt.Sprintf("%s-%040x.gosh", prefix, sha1.Sum(data))
internal/golden/golden.go:50: G304: Potential file inclusion via variable (gosec)
		tokens, err := ioutil.ReadFile(f + ".tokens")
main.go:135:21: Error return value of `liner.ReadHistory` is not checked (errcheck)
			liner.ReadHistory(f)
			                 ^
main.go:136:11: Error return value of `f.Close` is not checked (errcheck)
			f.Close()
			       ^
main.go:147:12: Error return value of `os.Create` is not checked (errcheck)
		os.Create(historyFilename)
		         ^
make: *** [check] Error 1
```

# What's the expected result?

No errors.

--- a/.golangci.yml
+++ b/.golangci.yml
@@ -33,5 +33,8 @@ linters:
 issues:
   exclude-use-default: false
   exclude:
-    # gas: Duplicates errcheck, but with more false positives like strings.Builder.WriteString
+    # gosec: Duplicates errcheck, but with more false positives like strings.Builder.WriteString
     - "G104: Errors unhandled"
+
+    # gosec: False positive triggered by `source, err := ioutil.ReadFile(f)`
+    - "G304: Potential file inclusion via variable"

--- a/internal/gofuzz/corpus.go
+++ b/internal/gofuzz/corpus.go
@@ -8,7 +8,7 @@
 package gofuzz
 
 import (
-	"crypto/sha1"
+	"crypto/sha1" //nolint:gosec // go-fuzz uses SHA1 for non-cryptographic hashing
 	"fmt"
 	"io/ioutil"
 	"os"
@@ -39,6 +39,7 @@ func AddFileToCorpus(name string, data []byte) {
 
 // AddDataToCorpus adds unnamed Gosh source code fragment to the go-fuzz corpus.
 func AddDataToCorpus(prefix string, data []byte) {
-	name := fmt.Sprintf("%s-%040x.gosh", prefix, sha1.Sum(data))
+	// go-fuzz uses SHA1 for non-cryptographic hashing
+	name := fmt.Sprintf("%s-%040x.gosh", prefix, sha1.Sum(data)) //nolint:gosec
 	AddFileToCorpus(name, data)
 }

--- a/internal/golden/golden.go
+++ b/internal/golden/golden.go
@@ -40,7 +40,7 @@ func init() {
 		panic(err)
 	}
 	for _, f := range files {
-		source, err := ioutil.ReadFile(f) //nolint:gas
+		source, err := ioutil.ReadFile(f)
 		if err != nil {
 			panic(err)
 		}

--- a/main.go
+++ b/main.go
@@ -54,7 +54,7 @@ func extractGoVersion(s string) string {
 }
 
 func goVersion() string {
-	cmd := exec.Command("go", "version") //nolint:gas
+	cmd := exec.Command("go", "version") //nolint:gosec
 	b, err := cmd.CombinedOutput()
 	if err != nil {
 		return ""

